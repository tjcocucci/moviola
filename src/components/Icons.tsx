export function MenuAlt2Icon() {
  return (
    <svg
      className="block w-6 text-gray-400"
      xmlns="http://www.w3.org/2000/svg"
      fill="none"
      viewBox="0 0 24 24"
      stroke="currentColor"
      strokeWidth={2}
    >
      <path
        strokeLinecap="round"
        strokeLinejoin="round"
        d="M4 6h16M4 12h16M4 18h16"
      />
    </svg>
  );
}

export function XIcon() {
  return (
    <svg
      className="block w-6 text-gray-400"
      xmlns="http://www.w3.org/2000/svg"
      fill="none"
      viewBox="0 0 24 24"
      stroke="currentColor"
      strokeWidth={2}
    >
      <path
        strokeLinecap="round"
        strokeLinejoin="round"
        d="M6 18L18 6M6 6l12 12"
      />
    </svg>
  );
}
