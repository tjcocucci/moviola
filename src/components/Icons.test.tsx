import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import { MenuAlt2Icon, XIcon } from './Icons';

describe('[ components / Icons ]', () => {
  describe('when component is rendered', () => {
    it('should render snapshoted Menu Icon', () => {
      const { asFragment } = render(<MenuAlt2Icon />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
  describe('when component is rendered', () => {
    it('should render snapshoted X icon', () => {
      const { asFragment } = render(<XIcon />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
