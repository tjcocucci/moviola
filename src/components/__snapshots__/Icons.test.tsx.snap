// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[ components / Icons ] > when component is rendered > should render snapshoted Menu Icon 1`] = `
<DocumentFragment>
  <svg
    class="block w-6 text-gray-400"
    fill="none"
    stroke="currentColor"
    stroke-width="2"
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M4 6h16M4 12h16M4 18h16"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</DocumentFragment>
`;

exports[`[ components / Icons ] > when component is rendered > should render snapshoted X icon 1`] = `
<DocumentFragment>
  <svg
    class="block w-6 text-gray-400"
    fill="none"
    stroke="currentColor"
    stroke-width="2"
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M6 18L18 6M6 6l12 12"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</DocumentFragment>
`;
